alibuild_helpers/doctor.py

```diff
@@ -72,31 +71,6 @@ def systemInfo():
     if not err:
       debug("%s:\n%s", f, out)
 
-def doctorArgParser(parser):
```

Comment:
This change is not related, is it?

Reply:
I moved that with the other code setting up args, so it's now at https://github.com/alisw/alibuild/pull/723/files/d3ba69b04105bea47cb62aba36e7eb53e344bb80#diff-0050151db7b4b98f46fb311611511578bee2f2a821a07665520e0e72e1fadb32L157 in alibuild_helpers/args.py (click "show diff"). This way, all the argparse code is together.